<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Linux Libertine Display G</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Linux Libertine Display G&quot;;</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>631</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.489, cy:0.551, radius:1.045, fx:1, fy:1, stop:1 rgba(162, 155, 162, 0));</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="mpty">
       <property name="styleSheet">
        <string notr="true">background-color: qradialgradient(spread:pad, cx:0.489, cy:0.551, radius:1.045, fx:1, fy:1, stop:1 rgba(162, 155, 162, 0));</string>
       </property>
      </widget>
      <widget class="QWidget" name="add_user">
       <property name="styleSheet">
        <string notr="true">background-color: qradialgradient(spread:pad, cx:0.489, cy:0.551, radius:1.045, fx:1, fy:1, stop:1 rgba(162, 155, 162, 0));</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>70</y>
          <width>261</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qradialgradient(spread:pad, cx:0.512, cy:0.573364, radius:1.045, fx:1, fy:1, stop:0 rgba(162, 155, 162, 0), stop:1 rgba(255, 255, 255, 255));
background-image: none;
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CONFERENTION NAME:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qradialgradient(spread:pad, cx:0.489, cy:0.551, radius:1.045, fx:1, fy:1, stop:0 rgba(162, 155, 162, 0), stop:1 rgba(255, 255, 255, 255));
background-image: none</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;PARTICIPANTS:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page"/>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>143</width>
        <height>501</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.489, cy:0.551, radius:1.045, fx:1, fy:1, stop:1 rgba(162, 155, 162, 0));
border: 1px solid;
border-color: qlineargradient(spread:pad, x1:0.074, y1:0.13, x2:1, y2:1, stop:0.0965909 rgba(0, 0, 0, 173), stop:0.352273 rgba(93, 93, 93, 232), stop:0.642045 rgba(214, 185, 235, 182), stop:0.977273 rgba(255, 255, 255, 173));</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>771</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid;
font: 12pt &quot;Linux Libertine Display G&quot;;
border-color: qlineargradient(spread:pad, x1:0.074, y1:0.13, x2:1, y2:1, stop:0.0965909 rgba(0, 0, 0, 173), stop:0.352273 rgba(93, 93, 93, 232), stop:0.642045 rgba(214, 185, 235, 182), stop:0.977273 rgba(255, 255, 255, 173));
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(255, 255, 255, 0), stop:0.373991 rgba(33, 30, 255, 0), stop:0.624043 rgba(255, 0, 0, 0), stop:1 rgba(255, 0, 0, 0));</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Linux Libertine Display G</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Friends&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>765</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add friend...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add friend...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid;
border-color: qlineargradient(spread:pad, x1:0.074, y1:0.13, x2:1, y2:1, stop:0.0965909 rgba(0, 0, 0, 173), stop:0.352273 rgba(93, 93, 93, 232), stop:0.642045 rgba(214, 185, 235, 182), stop:0.977273 rgba(255, 255, 255, 173));
background-color: qradialgradient(spread:pad, cx:0.512, cy:0.573364, radius:1.045, fx:1, fy:1, stop:0 rgba(162, 155, 162, 0), stop:1 rgba(255, 255, 255, 255));
background-image: none;
</string>
      </property>
      <property name="text">
       <string>Add friend...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/out.png</normaloff>:/images/out.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionfriends"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add New User</string>
   </property>
  </action>
  <action name="actionfriends">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/friends.png</normaloff>:/images/friends.png</iconset>
   </property>
   <property name="text">
    <string>friends</string>
   </property>
   <property name="toolTip">
    <string>Friends</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
